etc/ProdOfflineConfiguration.py

```diff
@@ -1174,6 +1167,21 @@
 ### Parking and Scouting PDs                         ###
 ########################################################
 
+DATASETS = ["ScoutingPFRun3"]
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=False,
+               scenario=ppScenario)
+
+
+DATASETS = ["ParkingDoubleElectronLowMass"]
```

Comment:
I think these should be ParkingDoubleElectronLowMass0-5

Reply:
Indeed, missed that problem - it won't work as is.

Reply:
Done with https://github.com/dmwm/T0/pull/4716/commits/c23749ff2d2b61a3e35899c8afc12f05aea15e4f